pipeline/02-assess.R

```diff
@@ -375,7 +375,11 @@ sales_data_two_most_recent <- sales_data %>%
   ) %>%
   # Include outliers, since these data are used for desk review and
   # not for modeling
-  rename(meta_sale_outlier_type = sv_outlier_type) %>%
+  rename(
+    meta_sale_outlier_reason1 = sv_outlier_reason1,
+    meta_sale_outlier_reason2 = sv_outlier_reason2,
+    meta_sale_outlier_reason3 = sv_outlier_reason3
+  ) %>%
   mutate(
     meta_sale_outlier_type = ifelse(
       meta_sale_outlier_type == "Not outlier", NA, meta_sale_outlier_type
```

Comment:
**issue (blocking):** This code removes the `meta_sale_outlier_type` column but keeps the very next statement where it's used (the `mutate()` call). `sv_outlier_type` is also referenced in a breaking way in the `distinct()` call right about this (L374).

Reply:
Do you know why the `build-and-run-model` check would have passed given the incorrect column references? @dfsnow